updates.jenkins.io.tf

```
@@ -0,0 +1,56 @@
+# Storage account
+resource "azurerm_resource_group" "updates_jenkins_io" {
+  name     = "updates-jenkins-io"
+  location = var.location
+  tags     = local.default_tags
+}
+
+resource "azurerm_storage_account" "updates_jenkins_io_files" {
+  name                          = "updatesjenkinsiofiles"
+  resource_group_name           = azurerm_resource_group.updates_jenkins_io.name
+  location                      = azurerm_resource_group.updates_jenkins_io.location
+  account_tier                  = "Standard"
+  account_replication_type      = "GRS"    # recommended for backups
+  min_tls_version               = "TLS1_2" # default value, needed for tfsec
+  public_network_access_enabled = "true"   # Explicit default value, we want this storage account to be readable from anywhere
+
+  tags = local.default_tags
+}
+
+output "updates_jenkins_io_files_primary_access_key" {
+  sensitive = true
+  value     = azurerm_storage_account.updates_jenkins_io_files.primary_access_key
+}
+
+# Redis database
+resource "azurerm_redis_cache" "updates_jenkins_io_database" {
```

```suggestion
resource "azurerm_redis_cache" "updates_jenkins_io" {
```

The name "database" is already implied by the fact it is a Redis resource